packages/manager/apps/telecom/src/app/telecom/telephony/svaWallet/kyc/identity/form/form.html

```diff
@@ -402,3 +431,48 @@ <h3 data-translate="telephony_billingAccount_svaWallet_kyc_identity_title"></h3>
         data-translate-attr="{ alt: 'telephony_billingAccount_svaWallet_kyc_identity_submit_lemonway' }"
     />
 </form>
+<div data-ng-if="$ctrl.isOpenModal">
+    <div class="modal d-block" tabindex="-1">
+        <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable">
+            <div class="modal-content">
+                <form novalidate name="modalConfirm">
+                    <oui-modal
```

Comment:
**question(non-blocking):** I'm wondering why we are not using a stateful modal here? Is there any reason?